src/index.js

```diff
@@ -39,6 +39,7 @@ export default function supernova(env) {
       const [expandedState, setExpandedState] = useState(null);
       const [linked, setLinked] = useState(false);
       const [selectionState, setSelectionState] = useState([]);
+      const [transform, setTransform] = useState(null);
```

Comment:
Would this be the same state as Elise has in her PR?

Reply:
Yes, Elise will merge first and then I'll see about the conflicts :)